### ISSUE TITLE
hard coded tomcat admin password in backend/pom.xml

### ISSUE THREAD

**Issue by sw360-blue-developer** [siemens/sw360portal#110]
```
siemagrant@vagrant-ubuntu-trusty-64:~$ fgrep -R sw360fossy /sw360portal/
/sw360portal/backend/pom.xml:                        <password>sw360fossy</password>
```
in order to be possible to change the tomcat password it should not be hardcoded in the sources

**Comment by maximilianhuber** [comment]
We are writing the password into the users `settings.xml` and that might be enough. Have you tried to simply remove the credentials from the `pom.xml`?

See also the [documentation](https://tomcat.apache.org/maven-plugin-2.0/tomcat7-maven-plugin/usage.html) of tomcat7-maven-plugin, where this way of configuration is also used.

**Comment by sw360-blue-developer** [comment]
simple removing is not working because the backend deployment uses the tomcat manager for the deployment.

It exists a second way to deploy the backend like the frontend by copying the war files into the liferay deploy folder by using the liferay-maven-plugin.

this has two preconditions:
1. the tomcat have to be shutdown (or not even started) before deploying because the liferay maven plugin is quiet slow on copying. the portal tries to unpack the war while the copy process and fails. this requires changing the sw360-install.sh of the vagrant project as well
2. the liferay-maven-plugin has to be upgrade in sw360/pom.xml from version 6.2.1 to 6.2.5 (released Jan 16) because 6.2.1 has a bug which do not close all file streams and fails later at copying the svc-attachments.war. I tested 6.2.5 successfully

**Comment by maximilianhuber** [comment]
Change the part
```                                                                                                                                                         
                     <configuration>                                                                                                                                                                
                         <server>${tomcat-backend-server}</server>                                                                                                                                                                                             
                         <path>/${project.build.finalName}</path>                                                                                                                                                                   
                         <url>${tomcat-backend-url}</url>                                                                                                                                                               
                         <username>admin</username>                                                                                                                                                                    
                         <password>sw360fossy</password>                                                                                                                                                                      
                     </configuration>
```
of the file `backend/pom.xml` to
```                                                                                                                                                         
                     <configuration>
                         <server>localtomcat</server>
                     </configuration>
```
The file `/etc/maven/settings.xml`, which is created from
 `puppet/modules/sw360/templates/maven_settings.xml.erb` already contains the credentials with the Id localtomcat.

(works for me: old vagrant without the PR nr. 39 but with manually added credentials to `/etc/maven/settings.xml`)
